### PR TITLE
Add benchto/trino integration test 

### PR DESCRIPTION
**Pull request by ppalucha**
Add non-mocked test of integration between benchto-driver, benchto-service and Trino.

### CODE REVIEW

File: benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                .withNetwork(network)
+                .withNetworkAliases("trino")
+                .withExposedPorts(8080)
+                .waitingFor(new HttpWaitStrategy()
+                        .forPort(8080)
+                        .forPath("/v1/info")
+                        .forStatusCode(200)
+                        .forResponsePredicate(response -> response.contains("\"starting\":false")));
+        trino.start();
+        // We sometimes get "No nodes available to run query"
```
Comment (sopel39) [inline benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java:119]: I wonder why we don't have such issue for `io.trino.tests.product.launcher.env.common.Standard#createPrestoContainer`.

Maybe we should actually execute `SELECT 1` instead?

File: benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                .withNetwork(network)
+                .withNetworkAliases("postgres");
+        postgres.start();
+        String jdbcUrl = format("jdbc:postgresql://%s:%d/%s?user=%s&password=%s",
+                "postgres",
+                postgres.getExposedPorts().get(0),
+                postgres.getDatabaseName(),
+                postgres.getUsername(),
+                postgres.getPassword());
+        service = new GenericContainer<>("prestodev/benchto-service:latest")
```
Comment (sopel39) [inline benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java:92]: will local build with `maven` always be latest?

File: benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    @Autowired
+    private BenchmarkProperties benchmarkProperties;
+
+    @Autowired
+    protected ApplicationContext context;
+
+    @BeforeClass
+    public static void setup()
+    {
+        Network network = Network.newNetwork();
```
Comment (sopel39) [inline benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java:67]: will this cleanup automatically after test is finished?

File: pom.xml [hunk 4]
```diff
@@ -56,13 +56,13 @@
         <dep.commons.math3>3.6.1</dep.commons.math3>
         <dep.commons.io>2.7</dep.commons.io>
         <dep.trino.jdbc.version>351</dep.trino.jdbc.version>
-        <dep.jackson.version>2.4.5</dep.jackson.version>
```
Comment (sopel39) [inline pom.xml:59]: updating deps should be separate commit

File: benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+    }
+
+    private void verifyBenchmark()
+    {
+        UriComponents url = UriComponentsBuilder.fromHttpUrl(getServiceUrl()).path("v1/benchmark/test_benchmark/BEN_SEQ_ID").build();
+        RestTemplate restTemplate = new RestTemplate();
+        ResponseEntity<String> response = restTemplate.getForEntity(url.toUri(), String.class);
+        DocumentContext json = JsonPath.parse(response.getBody());
+        assertThat(json.read("$.status", String.class)).isEqualTo("ENDED");
+        assertThat(json.read("$.executions[?(@.status == 'ENDED')]", List.class)).hasSize(2);
```
Comment (sopel39) [inline benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java:159]: it it possible that to check that there were not errors during query execytion?

File: benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java (hunk 2 above)
Comment (ppalucha) [reply benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java:92]: Yes, maven docker plugin by default tags it with 'latest'.

File: benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java (hunk 3 above)
Comment (ppalucha) [reply benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java:67]: Yes, I see no networks left after tests execution.

File: benchto-driver/src/main/java/io/trino/benchto/driver/macro/query/QueryMacroExecutionDriver.java [hunk 6]
```diff
@@ -86,7 +86,9 @@ public void runBenchmarkMacro(String macroName, Optional<Benchmark> benchmarkOpt
             }
         }
         catch (SQLException e) {
-            throw new BenchmarkExecutionException("Could not execute macro SQL queries for benchmark: " + benchmark, e);
+            throw new BenchmarkExecutionException(
+                    "Could not execute macro SQL queries for benchmark: " + benchmark + " (" + e.getMessage() + ")",
+                    e);
```
[the remark this answers is not in the document]
Comment (ppalucha) [reply benchto-driver/src/main/java/io/trino/benchto/driver/macro/query/QueryMacroExecutionDriver.java:91]: Fixed at the source (while creating message for benchmark failure cause)
